index.js

```diff
@@ -1,39 +1,99 @@
 async function setupPlugin({config, global}) {
-    global.posthogUrl = config.postHogUrl
+    global.posthogUrl = cleanInstanceUrl(config.postHogUrl)
     global.apiToken = config.postHogApiToken
     global.projectToken = config.postHogProjectToken
-    global.syncScoresIntoPosthog = global.posthogUrl && global.apiToken && global.projectToken
+    global.zapierUrl = config.zapierUrl
+    global.actionIdToName = {}
+    config.actionIdToName.split(',').forEach(pair => {
+        const [actionId, conversionName] = pair.split(':')
+        global.actionIdToName[parseInt(actionId)] = conversionName
+    })
+    global.defaultStartTime = config.defaultStartTime ? new Date(config.defaultStartTime) : new Date(2021, 6, 1)
```

Comment:
I don't think we need this anymore since we're setting this default in the Json file